packages/components/src/utils/style-mixins.js

```diff
@@ -3,5 +3,6 @@ export { reduceMotion } from './reduce-motion';
 export { rtl } from './rtl';
 export { font } from './font';
 export { breakpoint } from './breakpoint';
+export { flow } from './flow';
```

Comment:
I wasn't too sure how widespread usage of the `interpolate` functions would be, so left it out to be consistent with the other math-related utils.

Instead of adding it to `style-mixins.js`, should we be exporting all math utils? If so, maybe a follow-up PR would be best?

Reply:
Makes sense, this is definitely a tidy-up task that can be done in a follow-up